### PR TITLE
Remove search button on Patients page

### LINKED ISSUE #3847: [BUG] Multiplex People Screen: Extraneous Button on Search Field
### Description
On the "People" screen, the search input field is attached to a blue "button" on the right. This button has a hover state but no label, and when clicked does not appear to have any impact on what is displayed on screen.

### Steps to reproduce

1. Go to the "People" screen
2. Begin typing in the search field. 
3. Click on the blue "button" next to the search field.
4. As you type a name into the Search field, the results automatically appear in the list on the screen.
5. Clicking on the blue button attached to the Search field does not affect the results displayed on screen.

### Expected behavior
As it currently exists, this button has no text label or function and provides no value to the user. 
This button should be removed.

### Additional context
Bug experienced on a MacBookPro running macOS Monterey Version 12.4
Chrome Version 102.0.5005.61 (Official Build) (x86_64)
Screenshot is attached.
people-search

### PR DESCRIPTION
**Pull request by emyl3**
# FRONTEND PULL REQUEST

## Related Issue
Resolves #3847

## Changes Proposed
Removes the search button on the "Patients" page because it was not needed to search for patients in the first place

## Testing
- Go to the "Patients" tab and search for a patient
- Search should work as intended

## Screenshots / Demos
**Before**
Screen Shot 2022-12-06 at 12 56 34

**After**
Screen Shot 2022-12-06 at 12 55 57

### CODE REVIEW

**Review by BobanL — APPROVED** [review]
👍

**Comment by nathancrtr** [comment]
FWIW, this will introduce an instance of a [sort-of bug](https://app.zenhub.com/workspaces/simplereport-better-data-team-605b43d51ea9f700119a48ee/issues/cdcgov/prime-simplereport/4695).

Prior to this branch, pressing `Enter` within the search field is a no-op. With the search button removed, an `Enter` keypress will attempt to submit the form and bring the user to the facility selection screen:

https://user-images.githubusercontent.com/27730981/206007983-6c4c4695-1bce-497c-8cbe-cb39d619ac63.mov

This is explained in fuller detail in #4695, which also addresses the issue.

I'd propose waiting to merge this until after #4695 is in to avoid this tricky regression, but otherwise this is great!

**Comment by emyl3** [comment]
> FWIW, this will introduce an instance of a [sort-of bug](https://app.zenhub.com/workspaces/simplereport-better-data-team-605b43d51ea9f700119a48ee/issues/cdcgov/prime-simplereport/4695).
> 
> Prior to this branch, pressing `Enter` within the search field is a no-op. With the search button removed, an `Enter` keypress will attempt to submit the form and bring the user to the facility selection screen:
> 
>  Screen.Recording.2022-12-06.at.2.46.28.PM.mov 
> This is explained in fuller detail in #4695, which also addresses the issue.
> 
> I'd propose waiting to merge this until after #4695 is in to avoid this tricky regression, but otherwise this is great!

Thank you for letting me know, Nathan! Will wait till that fix is in place!

**Review by nathancrtr — APPROVED** [review]
We should double-check once `main` is merged into this branch, but https://github.com/CDCgov/prime-simplereport/issues/4695 has been merged so this one should be good to go!

**Comment by emyl3** [comment]
> We should double-check once `main` is merged into this branch, but #4695 has been merged so this one should be good to go!

Thank you for the update, @nathancrtr 😸  I'm going pull main in and test this branch out now! :D

**Comment by emyl3** [comment]
@BobanL @nathancrtr Ready for re-review :) Worked as intended following merge of #4695

**Review by nathancrtr — APPROVED** [review]

**Review by mehansen — APPROVED** [review]
🚢

**Pull request merged**
